src/version.js

```diff
@@ -1,4 +1,4 @@
-const version = '{{versionString}}';
+const version = process.env.VERSION; // eslint-disable-line
```

Comment:
It's prebuilt babel plugin. [babel-plugin-transform-inline-environment-variables](https://www.npmjs.com/package/babel-plugin-transform-inline-environment-variables)

This is a pretty common thing to do in Browserify land. It's useful if you want to do something like: 
```
if (process.env.NODE_ENV === 'development') { do something just for testing }
```

Or another use case for us would be if we wanted to do something like.

```
if (process.env.ENVIRONMENT === 'node') { do something just for the node build }
```